llama_stack/cli/stack/stack.py

```diff
@@ -24,6 +25,12 @@ def __init__(self, subparsers: argparse._SubParsersAction):
             description="Operations for the Llama Stack / Distributions",
         )
 
+        self.parser.add_argument(
+            "--version",
+            action="version",
+            version=f"llama-stack {version('llama-stack')}",
```

Comment:
it should only output the version not the `llama-stack ` prefix

Reply:
done